man/systemd.netdev.xml

```diff
@@ -911,6 +911,13 @@
           <para>Accepts the same key as <literal>[FooOverUDP]</literal></para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>6rdPrefix=</varname></term>
```

Comment:
I do not like the name of this setting. What `6rd` does stand for?

Reply:
It is not strong opinion, but I prefer `IPv6RapidDeploymentPrefix=`. Maybe mentioning '6rd' in the explanation below is better, something like 'This enables IPv6 Rapid Deployment, known as 6rd.'